ninja-maven-plugin/src/test/java/ninja/build/WatchAndRestartMachineTest.java

```diff
@@ -217,5 +232,63 @@ public void testCheckIfMatchesPatternAndFtlHtmlFilesAreIgnored() {
  
         
     }
+
+        @Rule public ExpectedException thrown = ExpectedException.none();
+    /* testedClasses: WatchAndRestartMachine */
+    /*
+     * Test generated by Diffblue Deeptest.
+     * This test case covers:
+     *  - conditional line 273 branch to line 273
+     */
+    @Test
+    public void checkIfWouldBeExcludedInput0NotNullOutputFalse() {
+      // Arrange
+      final java.util.HashSet<String> patterns = new java.util.HashSet<String>();
+      final String string = "";
+      // Act
+      final boolean retval = WatchAndRestartMachine.checkIfWouldBeExcluded(patterns, string);
+      // Assert result
+      Assert.assertEquals(false, retval);
+    }
+    /*
+     * Test generated by Diffblue Deeptest.
+     * This test case covers the entire method.
+     */
+    @Test
+    public void constructorInputNotNullNotNullFalseOutputVoid() {
```

Comment:
discard

Reply:
Discard. Method under test is a constructor.